scripts/modelbench/inductor_test.sh

```diff
@@ -10,14 +12,50 @@ DT=${3:-float32}
 # static / dynamic
 SHAPE=${4:-static}
 LOG_DIR=${5:-inductor_log}
+DYNAMO_BENCH=${6:-1238ae3}
+# default / cpp
+WRAPPER=${7:-default}
+
 mkdir -p $LOG_DIR
 
+
+# https://github.com/pytorch/pytorch/issues/107200
+pip uninstall transformers -y && pip install transformers==4.30.2
+# collect sw info
+curdir=$(pwd)
+touch ${curdir}/${LOG_DIR}/version.txt
+cd /workspace/benchmark
+echo torchbench : $(git rev-parse --short HEAD) >>${curdir}/${LOG_DIR}/version.txt
+cd /workspace/pytorch
+python -c '''import torch,torchvision,torchtext,torchaudio,torchdata; \
+	print("torch : ", torch.__version__); \
+	print("torchvision : ", torchvision.__version__); \
+	print("torchtext : ", torchtext.__version__); \
+	print("torchaudio : ", torchaudio.__version__); \
+	print("torchdata : ", torchdata.__version__)''' >>${curdir}/${LOG_DIR}/version.txt
+echo dynamo_benchmarks : $DYNAMO_BENCH >>${curdir}/${LOG_DIR}/version.txt
+
```

Comment:
How about to move out those version part both in inference and training test scripts as a standalone scripts?

Reply:
moved to **version_collect.sh**